frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.tsx

```diff
@@ -155,7 +153,8 @@ export const StaticEmbedSetupPane = ({
     ],
   );
 
-  const iframeUrl = iframeUrlWithoutHash + optionsToHashParams(displayOptions);
+  const iframeUrl =
```

Comment:
This fix the minor differences when generating the iframe preview URL vs the actual code in the code panel.

This doesn't make any difference in practice, but I use the iframe code for testing this feature, so it's nice to make them the same.

Previously, some default options where always in the preview URL even though we don't show it in the code e.g. `background=true` won't be in the url since it's a default value.